contract/src/main/java/com/ritense/valtimo/contract/security/jwt/TokenAuthenticator.java

```diff
@@ -19,10 +19,10 @@
 import io.jsonwebtoken.Claims;
 import org.springframework.security.core.Authentication;
 
-public abstract class TokenAuthenticator {
+public interface TokenAuthenticator {
```

Comment:
Don't forget to also document this as a breaking change

Reply:
I made a not of it in the story.